README.md

```diff
@@ -1 +1,65 @@
-# capstone_1
\ No newline at end of file
+# capstone_1
+
+This project is about creating a mobile and desktop view of my personal portifolio that is assigned by microverse. 
+
```

Comment:
Please, make sure that your README file contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) used. This is required by the [Creative Commons License](https://creativecommons.org/licenses/) under which it is licensed.

Reply:
Added Acknowledgements to the README